slackbot_settings.py

```diff
@@ -0,0 +1,15 @@
+import dotenv
+
+dotenv.load()
+
+API_TOKEN = dotenv.get('SLACKBOT_API_TOKEN')
+DEFAULT_REPLY = 'Sorry, I have not been fully configured yet.'
+
+# TO DO:
+# The value for ERRORS_TO should be specified in the env file later on
```

Comment:
Because the channel it is going to at the moment is hardcoded. It would make more sense having it in an ENV file so we can switch channels easily without having to update the code all the time.